rpcs3/Loader/TROPUSR.cpp

```diff
@@ -5,16 +5,24 @@
 
 LOG_CHANNEL(trp_log, "Trophy");
 
+enum : u32
+{
+	TROPUSR_MAGIC = 0x818F54AD
+};
+
 bool TROPUSRLoader::Load(const std::string& filepath, const std::string& configpath)
 {
 	const std::string& path = vfs::get(filepath);
 
-	if (!m_file.open(path, fs::read))
+	if (!m_file.open(path))
 	{
+		// Generate TROPUSR.DAT
 		if (!Generate(filepath, configpath))
 		{
 			return false;
 		}
+
+		return true;
```

Comment:
Why is it changed to true?

Reply:
Generated TROPUSR.DAT successfuly, no error here.

Reply:
then simplify to:
`return Generate(filepath, configpath);`